tools/cellprofiler_v4/cellprofiler.xml

```diff
@@ -0,0 +1,244 @@
+<tool id="cp_cellprofiler" name="CellProfiler" version="@CP_VERSION@+galaxy@VERSION_SUFFIX@">
+    <description>run a CellProfiler pipeline</description>
+    <macros>
+        <import>macros.xml</import>
+        <token name="@VERSION_SUFFIX@">0</token>
+        <xml name="test_assert_content" token_n="291">
+            <assert_contents>
+                <has_n_lines n="@N@" />
+            </assert_contents>
+        </xml>
+    </macros>
+    <expand macro="cp_requirements" />
+    <command detect_errors="aggressive"><![CDATA[
+        bash '$script_file' &&
```

Comment:
```suggestion
        export MPLCONFIGDIR=\$TEMP &&
        bash '$script_file' &&
```

Reply:
It fixes the matplotlib problem, but there is still a problem with some dependencies.